src/main/java/net/minecraftforge/client/extensions/common/IClientBlockExtensions.java

```diff
@@ -14,15 +14,32 @@
 import net.minecraft.world.item.enchantment.EnchantmentHelper;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.LevelReader;
+import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.level.material.Material;
 import net.minecraft.world.phys.HitResult;
+import net.minecraftforge.fml.LogicalSide;
 
-public interface IBlockRenderProperties
+import java.util.function.Consumer;
+
+/**
+ * {@link LogicalSide#CLIENT Client-only} extensions to {@link Block}.
```

Comment:
The label should be plain text, so the tag should be `{@linkplain}`.
```suggestion
 * {@linkplain LogicalSide#CLIENT Client-only} extensions to {@link Block}.
```